### PR TITLE
TF-1330 Fix can not swipe left/right in email view on mobile

### PR DESCRIPTION
**Pull request by dab246**
#1330 

### Resolved

- Android Mobile

https://user-images.githubusercontent.com/80730648/208636942-0e415cf8-89d1-4f4e-9f92-9a99e581c8f0.mov

- iOS Mobile

https://user-images.githubusercontent.com/80730648/208637021-da52c640-0555-4809-8a1a-755b22ca9835.mp4

- iOS iPad

https://user-images.githubusercontent.com/80730648/208637105-afe41ee4-e81e-4d1c-88d7-5083d1c381ff.mov

### CODE REVIEW

**Review by hoangdat — APPROVED** [review]

**Review by ManhNTX — APPROVED** [review]
LGTM

File: core/lib/presentation/views/html_viewer/html_content_viewer_widget.dart [hunk 1]
```diff
@@ -183,6 +184,18 @@ class _HtmlContentViewState extends State<HtmlContentViewer> {
       if (scrollWidth != null && offsetWidth != null && mounted) {
         final isScrollActivated = scrollWidth.round() == offsetWidth.round();
         log('_HtmlContentViewState::_setActualWidthView():isScrollActivated: $isScrollActivated');
+        if (isScrollActivated) {
```
Comment (sherlockvn) [inline core/lib/presentation/views/html_viewer/html_content_viewer_widget.dart:187]: i think `horizontalGestureActivated = !isScrollActivated;` is more readable.

File: lib/features/email/presentation/controller/email_supervisor_controller.dart [hunk 2]
```diff
@@ -61,13 +61,11 @@ class EmailSupervisorController extends GetxController {
   void createPageControllerAndJumpToEmailById(EmailId currentEmailId) {
     _currentEmailIndex = _currentListEmail.matchedIndex(currentEmailId);
     if (pageController != null && pageController?.hasClients == true) {
-      _jumpToPage(_currentEmailIndex);
+      pageController?.jumpToPage(_currentEmailIndex);
```
Comment (sherlockvn) [inline lib/features/email/presentation/controller/email_supervisor_controller.dart:64]: for sure, pageController will not be null, so why do you use ? after pageController

**Review by sherlockvn — APPROVED** [review]

File: lib/features/email/presentation/controller/email_supervisor_controller.dart (hunk 2 above)
Comment (dab246) [reply lib/features/email/presentation/controller/email_supervisor_controller.dart:64]: We can write in 2 ways it doesn't affect anything.

File: core/lib/presentation/views/html_viewer/html_content_viewer_widget.dart (hunk 1 above)
Comment (dab246) [reply core/lib/presentation/views/html_viewer/html_content_viewer_widget.dart:187]: I also perform another action in this condition. You can write as you please, but we'll need to write more conditions to handle the next action.

File: lib/features/email/presentation/controller/email_supervisor_controller.dart (hunk 2 above)
Comment (hoangdat) [reply lib/features/email/presentation/controller/email_supervisor_controller.dart:64]: :)

**Pull request merged**